examples/thread_test/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "thread_test"
+authors = ["Marco Ruppert-Maas <marco.ruppert-maas@studium.fernuni-hagen.de>"]
+edition = "2021"
+
+[target.'cfg(target_os = "hermit")'.dependencies]
+hermit = { path = "../../hermit", default-features = false }
+
+[dependencies]
+log = "0.4"
+simple_logger = "4.0"
```

Comment:
Since we are just doing `log::info!` here, can we just replace them with `println!` to keep things simpler?

```suggestion
```